app/models/project/Page.scala

```diff
@@ -141,7 +182,10 @@ object Page {
         TypographicExtension.create()
       ))
 
-    (Parser.builder(options).build(), HtmlRenderer.builder(options).build())
+    (Parser.builder(options).build(), HtmlRenderer.builder(options)
+      // TODO: Get OreConfig using dependency injection or pass as parameter
+      .linkResolverFactory(new ExternalLinkResolver.Factory(play.api.Play.current.injector.instanceOf(classOf[OreConfig])))
```

Comment:
Not sure how to get an instance of  `OreConfig` here. Maybe pass as implicit parameter to `Render(String)` below and build a new `htmlRenderer` with the config?

Reply:
See above